src/sentry/static/sentry/app/components/idBadge/organizationBadge.tsx

```diff
@@ -0,0 +1,54 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import BaseBadge from 'app/components/idBadge/baseBadge';
+import BadgeDisplayName from 'app/components/idBadge/badgeDisplayName';
+import SentryTypes from 'app/sentryTypes';
+import {Organization} from 'app/types';
+import OrganizationAvatar from 'app/components/avatar/organizationAvatar';
+import withOrganization from 'app/utils/withOrganization';
+
+type Props = {
+  // A full organization is not used, but required to satisfy types with
+  // withOrganization()
+  organization: Organization;
+  className?: string;
+  avatarSize?: OrganizationAvatar['props']['size'];
+  // If true, will use default max-width, or specify one as a string
+  hideOverflow?: boolean | string;
+  hideAvatar?: boolean;
+};
+
+class OrganizationBadge extends React.Component<Props> {
+  static propTypes = {
+    ...BaseBadge.propTypes,
```

Comment:
Do we need a `TODO(ts)` for when we convert `BaseBadge`?

Reply:
I'm going to try and do that whole module tree as my next block of typescript updates.